### PR TITLE
adding minDate & maxDate optional attrs

### PR DESCRIPTION
**Pull request by IbrahimHd**
I need your help in warping the `ng-disabled`

Please have a look at my attempt and do you review.
Thnx

### CODE REVIEW

**Comment by g1eb** [comment]
Sure thing, let me take a look :)

Can you elaborate on what you want to achieve? Do you want to add minDate and maxDate as boundaries and disable input that is outside the allowed range?

I think it's a would be a useful addition and I'm going to add this to other inputs as well :)

File: src/date-input.html [hunk 1]
```diff
@@ -28,6 +27,7 @@
               'active': calendar.clone().startOf('month').add($parent.$index, 'weeks').weekday($index).startOf('day').isSame(selected.clone().startOf('day')),
               'inactive': calendar.clone().startOf('month').add($parent.$index, 'weeks').weekday($index).month() !== calendar.month()
             }"
+            ng-disabled="(!!minDate && calendar.clone().startOf('month').add($parent.$index, 'weeks').weekday($index).date().isBefore(minDate.date())) || (!!maxDate && calendar.clone().startOf('month').add($parent.$index, 'weeks').weekday($index).date().isAfter(maxDate.date()))"
```
Comment (IbrahimHd) [inline src/date-input.html:30]: I'm trying to put a boundaries on the Calendar based on minDate & maxDate attrs.
I'm using `isBefore()` & `isAfter() ` monmentJS comparison functions.
My doubt is about this part:
`calendar.clone().startOf('month').add($parent.$index, 'weeks').weekday($index).date()`
which I took it from ng-bind.
Plz feel free to do your **corrections** as necessary.

**Comment by g1eb** [comment]
Hi! I've fixed this merge and added a small visual indication of when selection is not allowed (similar in style to the ```datetime-range``` directive)

v0.1.24

**Comment by IbrahimHd** [comment]
I want to draw your kind attention that the piece of code that I proposed:

`ng-disabled="(!!minDate && calendar.clone().startOf('month').add($parent.$index, 'weeks').weekday($index).date().isBefore(minDate.date())) || (!!maxDate && calendar.clone().startOf('month').add($parent.$index, 'weeks').weekday($index).date().isAfter(maxDate.date()))"`

still needs some ratification, as it's not functioning as expected at my side.
did the `ng-disabled` work as expected at your side? if not, could you plz fix it?

**Comment by g1eb** [comment]
Oh yeah, my bad it was still in there.. ```ng-disabled``` only works on input elements that work with ```disabled``` attribute, it had no function here so I removed it 😄 

v0.1.25

**Pull request merged**